### ISSUE TITLE
Bug: Batch Subtitles Checkbox Enable/Disable Issue

### ISSUE THREAD

**Issue by GiBiT** [eibol/ffmpeg_batch#104]
Repeatable Steps:

1. Click Batch Subtitles Tab  
2. Highlight a File in the List
3. Click "Use input file subtitle stream" to turn on the checkbox
4. While that's checked "on" Click on "Burn Subtitles" and now both are active
5. Now Uncheck "Burn Subtitles" -- Now notice that "Use input file subtitle stream" is disabled. 

You can no longer replicate the initial functionality where both checkboxes are not checked and both are not disabled. "Burn subtitles" MUST be checked to disable "Use input file subtitle stream" 

So either the issue is initially "Use input subtitle stream" should be disabled so that "Burn subtitles" must be clicked, or there is a minor bug with the interactions between the checkboxes and how they're disabling each other.

**Comment by GiBiT** [comment]
Sorry forgot this: Version 3.0.5

**Comment by eibol** [comment]
HI, may I know the case scenario in which you need to use the input file subtitle stream but without burning (re-encoding) it?

**Comment by GiBiT** [comment]
I'm not sure of the scenario. I just upgraded to from an older version and this new checkbox was here, so I was clicking around to see what it did. I just noticed the difference between initially being able to click both but not once I click Burn Subtitles. 

I guess it's just a front end thing and may not effect backend? I'm not sure, but if the "Use input file subtitle stream" checkbox is checked and burn subtitles is not, "Use input file subtitle stream" stays checked but is disabled. I don't know if when you run it, it ignores the disabled checkbox or not.
